bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/wizard/NewIDFProjectWizard.java

```diff
@@ -165,12 +168,13 @@ private void createDefaultDebugConfig()
 
 		PageChangingEvent pageChangingEvent = new PageChangingEvent(wizard, wizard.getStartingPage(), editPage);
 		editPage.handlePageChanging(pageChangingEvent);
-
 		wizard.performFinish();
 
 		try
 		{
-			wizard.getWorkingCopy().doSave();
+			String originalName = wizard.getWorkingCopy().getName();
+			String debugConfigName = originalName.substring(0, originalName.lastIndexOf("Configuration")) + " Debug"; //$NON-NLS-1$ //$NON-NLS-2$
```

Comment:
We can remove the space before the " Debug" here as this will add the double space considering originalName.substring(0, originalName.lastIndexOf("Configuration")) will still return the project name ends with space after removing "Configuration" from the originalName